src/SFML/Graphics/Texture.cpp

```diff
@@ -742,7 +742,7 @@ void Texture::bind(const Texture* texture, CoordinateType coordinateType)
         glCheck(glBindTexture(GL_TEXTURE_2D, texture->m_texture));
 
         // Check if we need to define a special texture matrix
-        if ((coordinateType == Pixels) || texture->m_pixelsFlipped)
+        if ((coordinateType == sf::Pixels) || texture->m_pixelsFlipped)
```

Comment:
In case we go for the typedef, this change could be reverted.

Reply:
Done